server/proxy/expand.go

```diff
@@ -249,7 +249,6 @@ func (e *expander) expandUser(userPtr **model.User, userID string) expandFunc {
 func (e *expander) expandApp(level apps.ExpandLevel) error {
 	e.ExpandedContext.App = e.app.Strip(level)
 
-	e.ExpandedContext.App.WebhookSecret = ""
 	if level == apps.ExpandAll && e.r.RequireSysadminOrPlugin() == nil {
 		e.ExpandedContext.App.WebhookSecret = e.app.WebhookSecret
```

Comment:
Can you please elaborate why this current logic is wrong? The webhook secret should only get send to system admins.

Reply:
Noted, but when expanding `app:all`, shouldn't that too get webhook secret ?

Reply:
Also it seems to working with the changes in this PR, I could not fix tests for it currently, but tested manually

Reply:
If a non system admin expands `app:all` they should not get the webhook secrets. Only admins should get it.

Reply:
Got it. Thank you.
I will try to look into it again!